### ISSUE TITLE
Complete readme part with codes

### ISSUE THREAD

**Issue by mathias22osterhagen22** [HrithikMittal/Loopback4-auth#1]
First of all, thank you again for your works.

Could it be possible to add more details with code example in the differents steps, please?

Like
# Step 3:
Create a validate service to check the isEmail and password length
```ts
import {HttpErrors} from '@loopback/rest';
import * as isEmail from 'isemail';
import {Credentials} from '../repositories/index';

export function validateCredentials(credentials: Credentials) {
  if (!isEmail.validate(credentials.email)) {
    throw new HttpErrors.UnprocessableEntity('invalid Email');
  }
  if (credentials.password.length < 8) {
    throw new HttpErrors.UnprocessableEntity('password length should be greater than 8')
  }
}
````

Or "Use `hashPassword` method" like where ? why ? how ? 

Or "Now just save the user into the db" where ? how ?

Thank you

**Comment by HrithikMittal** [comment]
Okay I will do it but can you just help me by telling on which steps you need more details and moreover please check my new repo also of authorization https://github.com/HrithikMittal/loopback4-authorization

**Comment by mathias22osterhagen22** [comment]
I think it's on all that you used only sentences without code below.
I'm trying to use loopback for the first time, do it like if you are speaking to the most retard man in your class.

Like really step by step that a blind people without hand could follow.

(Yes I checked it but for now, I didn't get time to try it. I will probably make you some feedback on this second repository during the next weeks/months)

Thank you again for everything, because of you I finally succeed to implement the authentification system!  👍
